src/list.tsx

```diff
@@ -478,7 +478,12 @@ const RowContent = (props: RowContentProps) => {
                     onPress={toggle}
                     dataAttributes={dataAttributes}
                 >
-                    <Control name={name} checked={isChecked} aria-labelledby={titleId} />
+                    <Control
+                        name={name}
+                        checked={isChecked}
+                        aria-labelledby={titleId}
+                        render={(check) => check}
```

Comment:
why is this needed?

Reply:
needed to not use the default render, that will add the -2 hack that you see above, to let the list component works as it was previously